docs/recommendations/RNA/splicing.md

```diff
@@ -28,7 +32,7 @@ Variants affecting RNA splicing result in either a [_deletion_](../deletion/) or
     * NC\_000023.11(NM\_004006.2):r.649\_650ins650-50\_650-1: as a consequence of an intron 7 variant (c.650-52_650-51del) a new stronger exon 8 splice acceptor site is created (position 650-51 / 650-50) and the intron 7 sequence from positions 650-50 to 650-1 is inserted in the transcript: alternative description LRG\_199t1:r.649\_650ins650-50\_650-1
     * NC\_000023.11(NM\_004006.2):r.831\_832ins831+1\_831+67: as a consequence of an intron 8 variant (c.831+71C>A) a new stronger exon 8 splice donor site is created (position 831+67 / 831+68) and the intron 8 sequence from positions 831+1 to 831+67 is inserted in the transcript: alternative description LRG\_199t1:r.831\_832ins831+1\_831+67    
     * NC\_000023.11(NM\_004006.2):r.649\_650ins650-1400\_650-1268: as a consequence of an intron 7 variant (c.650-1401T>G) a new exon is created and its sequence (positions 650-1400 to 650-1268) is inserted in the transcript: alternative description LRG\_199t1:r.649\_650ins650-1400\_650-1268    
-* **fusion transcript** (based on [_SVD-WG007_](../../../consultation/SVD-WG007/)) 
+* **adjoined transcript segments** (based on [_SVD-WG007_](../../../consultation/SVD-WG007/) and subsequent alignment with the [VICC Gene Fusion Specification](https://fusions.cancervariants.org/en/latest) terminology) 
```

Comment:
per @jtdendunnen:
> why "adjoined transcript segments", we decided on "adjoined transcript"

Reply:
In short, adjoined transcripts are comprised of transcript segments, and “adjoined transcript segments” seemed like the clearer way of stating this. But I have no qualms about titling it “adjoined transcripts” and will make this change.

Reply:
addressed in 45493ca